components/faq/faq.tsx

```diff
@@ -0,0 +1,154 @@
+    }
+  };
+
+  useEffect(() => {
+    setAreAllExpanded(expandedArray.every((item) => item === true));
+  }, [expandedArray]);
+
+  return (
+    <section className={style.outer} id="faq">
+      <div className={style.inner}>
+        <div className={style.sticky}>
+          <h3>FAQ&apos;s</h3>
+          <p>
+            You can find the{' '}
+            <a href="https://slack.kedro.org/" target="_blank" rel="noreferrer">
+              Kedro community on Slack
+            </a>
+            .
+          </p>
+          <p>
+            We also maintain a list of articles, podcasts, talks, and Kedro
+            showcase projects in the{' '}
+            <a
+              href="https://github.com/kedro-org/kedro-community"
+              target="_blank"
+              rel="noreferrer"
+            >
+              kedro-community
+            </a>{' '}
+            repository.
+          </p>
+        </div>
+        <div className={style.questions}>
+          <div className={style.toggleAll} onClick={toggleExpandAll}>
+            {areAllExpanded ? 'Collapse all' : 'Expand all'}
+          </div>
+          {faqContent.map((topic, index) => (
+            <>
+              <hr />
+              <div className={style.accordionWrapper} key={topic.question}>
+                <Accordion
+                  index={index}
+                  isExpanded={expandedArray[index]}
+                  title={topic.question}
+                  toggleAccordion={toggleAccordion}
+                >
+                  <div className={style.answer}>
+                    <ReactMarkdown
+                      children={topic.answer}
+                      linkTarget="_blank"
+                    />
+                  </div>
+                </Accordion>
+              </div>
+            </>
+          ))}
+        </div>
+      </div>
+    </section>
+  );
+}
+
+type AccordionProps = {
+  children: JSX.Element;
+  index: number;
+  isExpanded: boolean;
+  title: string;
+  toggleAccordion: (index: number) => void;
+};
+
+const Accordion = ({
+  children,
+  index,
+  isExpanded,
+  title,
+  toggleAccordion,
+}: AccordionProps) => {
+  return (
+    <>
+      <motion.div
+        className={style.accordionHeader}
+        initial={false}
+        onClick={() => {
+          toggleAccordion(index);
+        }}
+      >
+        <h4>{title}</h4>
+        <svg
+          fill="none"
+          height="20"
+          style={{
+            transform: isExpanded ? 'rotate(0deg)' : 'rotate(180deg)',
+            transition: 'transform 0.175s ease',
+          }}
+          viewBox="0 0 16 20"
+          width="16"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path
+            clipRule="evenodd"
+            d="M0 8L8 0L16 8L14.5 9.5L9 4L9 20H7L7 4L1.5 9.5L0 8Z"
+            fillRule="evenodd"
+          />
+        </svg>
+      </motion.div>
+      <AnimatePresence initial={false}>
```

Comment:
this is very cool, I've never used it before.  I didn't know we use this for the website 😄

Reply:
framer-motion you mean?